editor/src/clj/editor/defold_project.clj

```diff
@@ -656,7 +670,18 @@
                  :resource-metrics @resource-metrics
                  :transaction-metrics @transaction-metrics})))))
 
+(defn reload-plugins! [project touched-resources]
+  (g/with-auto-evaluation-context evaluation-context
+    (let [workspace (workspace project evaluation-context)
+          code-preprocessors (workspace/code-preprocessors workspace evaluation-context)
+          code-transpilers (code-transpilers project)]
+      (workspace/unpack-editor-plugins! workspace touched-resources)
+      (code.preprocessors/reload-lua-preprocessors! code-preprocessors workspace/class-loader)
+      (code.transpilers/reload-lua-transpilers! code-transpilers workspace workspace/class-loader)
+      (workspace/load-clojure-editor-plugins! workspace touched-resources))))
```

Comment:
I moved plugin reload from workspace to project because it's needed for transpilers.